package-lock.json

```diff
@@ -0,0 +1,4613 @@
+{
```

Comment:
I've bootstrapped the project using `yarn` but for me, we can also use `npm`, the only thing is that we shouldn't use both

Reply:
Oh sorry, I'll fix that ;)

Reply:
No problem, if you prefer one over the other we can still change, I don't have strong arguments why one is better the other